src/odemis/acq/test/acq_stream_settings_test.py

```diff
@@ -0,0 +1,93 @@
+
+from odemis.acq.acq_stream_settings import AcquiredStreamSettings, MAX_LENGTH
+
+logging.getLogger().setLevel(logging.DEBUG)
+
+
+class TestAcquiredStreamSettings(unittest.TestCase):
+    """
+    Test the json file for reading and writing the most recently used acquired stream settings
+    """
+    # 11 entries and two duplication of the key "name"
+    test_data = [{'name': 'Filtered Colour 1', 'excitation': [0, 1, 2],
+                  'power': 0.0000000012, 'emission': [0.00001, 1.234e-07],
+                  'detExposureTime': 0.00123456789, 'auto_bc_outliers': 0.12345},
+                 {'name': 'Filtered Colour 2', 'excitation': [0, 1, 2],
+                  'power': 0.0000000034, 'emission': [0.00001, 1.234e-07],
+                  'detExposureTime': 0.00123456789, 'auto_bc_outliers': 0.12345},
+                 {'name': 'Filtered Colour 3', 'excitation': [0, 1, 2],
+                  'power': 0.0000000045, 'emission': [0.00001, 1.234e-07],
+                  'detExposureTime': 0.00123456789, 'auto_bc_outliers': 0.12345},
+                 {'name': 'Filtered Colour 4', 'excitation': [0, 1, 2],
+                  'power': 0.0000000056, 'emission': [0.00001, 1.234e-07],
+                  'detExposureTime': 0.00123456789, 'auto_bc_outliers': 0.12345},
+                 {'name': 'Filtered Colour 5', 'excitation': [0, 1, 2],
+                  'power': 0.0000000067, 'emission': [0.00001, 1.234e-07],
+                  'detExposureTime': 0.00123456789, 'auto_bc_outliers': 0.12345},
+                 {'name': 'Filtered Colour 6', 'excitation': [0, 1, 2],
+                  'power': 0.0000000078, 'emission': [0.00001, 1.234e-07],
+                  'detExposureTime': 0.00123456789, 'auto_bc_outliers': 0.12345},
+                 {'name': 'Filtered Colour 7', 'excitation': [0, 1, 2],
+                  'power': 0.0000000091, 'emission': [0.00001, 1.234e-07],
+                  'detExposureTime': 0.00123456789, 'auto_bc_outliers': 0.12345},
+                 {'name': 'Filtered Colour 8', 'excitation': [0, 1, 2],
+                  'power': 0.0000000092, 'emission': [0.00001, 1.234e-07],
+                  'detExposureTime': 0.00123456789, 'auto_bc_outliers': 0.12345},
+                 {'name': 'Filtered Colour 9', 'excitation': [0, 1, 2],
+                  'power': 0.0000000093, 'emission': [0.00001, 1.234e-07],
+                  'detExposureTime': 0.00123456789, 'auto_bc_outliers': 0.12345},
+                 {'name': 'Filtered Colour 10', 'excitation': [0, 1, 2],
+                  'power': 0.0000000094, 'emission': [0.00001, 1.234e-07],
+                  'detExposureTime': 0.00123456789, 'auto_bc_outliers': 0.12345},
+                 # Add tint
+                 {'name': 'Filtered Colour 1', 'excitation': [0, 1, 2],
+                  'power': 0.0000000095, 'emission': [0.00001, 1.234e-07], "tint": "red",
+                  'detExposureTime': 0.00123456789, 'auto_bc_outliers': 0.12345}]
+
+    @classmethod
+    def setUpClass(cls):
```

Comment:
Once you have functions in the `stream_settings` module to get/set the settings of one stream, you should add test cases for these functions here too.